### ISSUE TITLE
Error code 101 , and error is null 

### ISSUE THREAD

**Issue by khataha92** [anjlab/android-inapp-billing-v3#320]
Hi sir,
I'm using this library for In-App Purchase, I tried to follow instructions as in the description you provided, but I faced an error after trying to purchase,
The method `onBillingError` is called with errorCode:101, error: null
I tried to run a VPN but I have the same issue.
I made sure that the method `isInitialized` returns true.
Any help ?

**Comment by kriti-1** [comment]
Hey, even i am facing same problem did you find any solution for this?

**Comment by khataha92** [comment]
@kriti-1 , yes, I changed the logged in email to another email from different country that supports in app purchase, and it worked well.

**Comment by kriti-1** [comment]
So as am in india  this mean in app purchase wont work for any indian account?

**Comment by khataha92** [comment]
@kriti-1 I don't have an idea if your country doesn't support in app purchase,
just try purchasing on any other app in play store that contain purchase and if it successful, then that is not your issue, and if you did not find any app that has in app purchase, then your country doesn't support in app purchase.
